project/Build.scala

```diff
@@ -94,11 +96,9 @@ object MacroGLBuild extends MechaRepoBuild {
         mechaNightlyKey.dependsOn(compile in Compile)
     )
   
-  lazy val macrogl: Project = Project(
-    "macrogl",
-    file("."),
-    settings = macroglSettings
-  )
+  lazy val macrogl: CrossProject = CrossProject("macrogl", file("."), CrossType.Full)
+    .settings(macroglSettings)
```

Comment:
Sorry, I misunderstood `CrossProject` - I thought this was necessary to support cross compilation, but I see this is just for ScalaJS.